src/twisted/conch/test/test_transport.py

```diff
@@ -797,6 +799,62 @@ def test_sendKexInitBlocksOthers(self):
         self.proto._newKeys()
         self.assertEqual(self.transport.value().count(b"foo"), 2)
 
+    def test_sendExtInfo(self):
+        """
+        Test that EXT_INFO messages are sent correctly.  See RFC 8308,
+        section 2.3.
+        """
+        self.proto._supportsExtensionNegotiation = True
+        self.proto.sendExtInfo(
+            [
+                (b"server-sig-algs", b"ssh-rsa,rsa-sha2-256"),
+                (b"elevation", b"d"),
+            ]
+        )
+        self.assertEqual(
+            self.packets,
+            [
+                (
+                    transport.MSG_EXT_INFO,
+                    b"\x00\x00\x00\x02"
+                    + common.NS(b"server-sig-algs")
+                    + common.NS(b"ssh-rsa,rsa-sha2-256")
+                    + common.NS(b"elevation")
+                    + common.NS(b"d"),
+                )
+            ],
+        )
+
+    def test_sendExtInfoUnsupported(self):
+        """
+        If the other end has not advertised support for extension
+        negotiation, no EXT_INFO message is sent.  See RFC 8308, section
+        2.2.
+        """
+        self.proto.sendExtInfo([(b"server-sig-algs", b"ssh-rsa,rsa-sha2-256")])
+        self.assertEqual(self.packets, [])
+
+    def test_EXT_INFO(self):
+        """
+        Test that EXT_INFO messages are received correctly.  See RFC 8308,
+        section 2.3.
```

Comment:
```suggestion
        Once MSG_EXT_INFO is received it will  store the extension information a mapping.
```

received correctly ... is a bit generic in the context of a test docstring :)